dataactcore/models/jobTrackerInterface.py

```diff
@@ -241,16 +240,20 @@ def getNumberOfRowsById(self,jobId):
         """ Get number of rows in file for job matching ID """
         return self.getJobById(jobId).number_of_rows
 
+    def getNumberOfValidRowsById(self, jobId):
+        """Get number of file's rows that passed validations."""
+        return self.getJobById(jobId).number_of_rows_valid
```

Comment:
Love the question re: `list_submissions`--thanks for looking after the big picture.

The new valid row count on the job table is only used by the unit tests currently (though might be handy when we do the agency dashboard). Unlike error counts we store in the error db, which are by field/error type, this number just gives a high-level count of file rows that passed validations.

Does that answer your question?